monkey/infection_monkey/control.py

```diff
@@ -81,7 +81,7 @@ def find_server(default_tunnel=None):
                 if ControlClient.proxies:
                     debug_message += " through proxies: %s" % ControlClient.proxies
                 LOG.debug(debug_message)
-                requests.get("https://%s/api?action=is-up" % (server,),  # noqa: DUO123
+                requests.get(f"https://{server}/api?action=is-up",  # noqa: DUO123
```

Comment:
What version of python are we baselined on? 3.6 or 3.7?

Reply:
3.7